concordium-sdk/src/main/java/com/concordium/sdk/responses/rewardstatus/RewardsOverview.java

```diff
@@ -0,0 +1,78 @@
+package com.concordium.sdk.responses.rewardstatus;
+
+import com.concordium.sdk.responses.ProtocolVersion;
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.transactions.CCDAmount;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.Data;
+import lombok.Getter;
+
+import java.util.Date;
+
+/**
+ * Parsed Rewards Overview from response of
```

Comment:
This type of documentation is not really helpful for the consumer of the SDK.
Instead the documentation should state what to expect from the type and not that it's a parsed version of something else. 

E.g. 
`Get an overview of the balance of special accounts in the given block and some meta information about the state of the chain.`

Reply:
have changed it  to `Information about total amount of CCD and the state of various administrative accounts.`